src/pages/patch/PatchModal.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from "react";
```

Comment:
This file name is a little vague since there will likely be more patch modals in the future. Can you call it something like PatchRestartModal?